packages/lambda-xyz/src/__test__/xyz.test.ts

```diff
@@ -85,7 +89,7 @@ o.spec('LambdaXyz', () => {
         const res = await handleRequest(request);
         o(res.status).equals(200);
         o(res.header('content-type')).equals('image/webp');
-        o(res.header('eTaG')).equals('w8RW7/QQDHDMiiNDlbACrigBcCEveyedjHn7QMX6kDg=');
+        o(res.header('eTaG')).equals('foo');
```

Comment:
You do not appear to be asserting that `generate` is being called with the correct arguments

Reply:
I am unsure how much we need to test, considering typescript is already checking that the correct values are passed in. Also the values looked to be completely mocked anyway.